docs/tutorials/api/datasets.ipynb

```diff
@@ -0,0 +1,877 @@
+{
```

Comment:
<p>Technically we have seen that there might be cases where a <code>MapDataset</code>  could have a <code>RegionGeom</code>.  </p><p>I would also stress that <code>SpectrumDataset</code>  has no <code>psf</code>  associated.</p><p> </p><p>Maybe an array with datasets as rows and data type (i.e. counts of flux), reduced IRFs present, geometry and stat could be helpful and graphical? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gammapy/gammapy/pull/3488/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>